src/app/Instance/InstancePage/ErrorHandlingTabContent.tsx

```diff
@@ -167,62 +155,32 @@ export const ErrorHandlingTabContent = ({
     apiError !== undefined;
 
   return (
-    <PageSection
-      className="instance-page__tabs-error-handling__section"
-      variant={PageSectionVariants.light}
-    >
-      <Stack hasGutter>
-        <StackItem>
-          <Split hasGutter>
-            <SplitItem isFilled>
-              <TextContent>
-                <Text
-                  component={TextVariants.h2}
-                  ouiaId="error-handling-section"
-                >
-                  {t("common.errorHandlingMethod")}
-                </Text>
-              </TextContent>
-            </SplitItem>
-            {!isEditing && (
-              <SplitItem>
-                <Button
-                  isAriaDisabled={editIsDisabled}
-                  ouiaId="edit"
-                  onClick={(): void => setIsEditing(true)}
-                >
-                  {t("common.edit")}
-                </Button>
-              </SplitItem>
-            )}
-          </Split>
-        </StackItem>
-        <StackItem>
-          {isEditing ? (
-            <ErrorHandlingEdit
-              getSchema={getSchemaByMethod}
-              isLoading={isUpdateBridgeLoading || isSchemaLoading}
-              method={bridge?.error_handler?.type}
-              onCancelEditing={onCancelEditing}
-              onSubmit={onErrorHandlingSubmit}
-              parameters={
-                bridge?.error_handler?.parameters as Record<string, unknown>
-              }
-              registerValidateParameters={registerValidateParameters}
-              apiError={apiError}
-            />
-          ) : (
-            <ErrorHandlingDetail
-              errorHandlingType={bridge?.error_handler?.type}
-              errorHandlingParameters={bridge?.error_handler?.parameters}
-              isBridgeLoading={isBridgeLoading}
-              isSchemaLoading={isSchemaLoading}
-              schema={currentSchema}
-              apiError={apiError}
-            />
-          )}
-        </StackItem>
-      </Stack>
-    </PageSection>
+    <>
+      {isEditing ? (
+        <ErrorHandlingEdit
+          getSchema={getSchemaByMethod}
+          isLoading={isUpdateBridgeLoading || isSchemaLoading}
+          method={bridge?.error_handler?.type}
+          onCancelEditing={onCancelEditing}
+          onSubmit={onErrorHandlingSubmit}
+          parameters={
+            bridge?.error_handler?.parameters as Record<string, unknown>
+          }
+          registerValidateParameters={registerValidateParameters}
+          apiError={apiError}
+        />
+      ) : (
+        <ErrorHandlingDetail
+          errorHandlingType={bridge?.error_handler?.type}
+          errorHandlingParameters={bridge?.error_handler?.parameters}
+          isBridgeLoading={isBridgeLoading}
+          isSchemaLoading={isSchemaLoading}
+          schema={currentSchema}
+          apiError={apiError}
+          onEdit={(): void => setIsEditing(true)}
+          isEditDisabled={editIsDisabled}
+        />
+      )}
```

Comment:
Separating the layout of the EH tab between the "view" and "edit" mode.
The PF `Stack` layout was mainly useful only to the view mode.